pandas/core/window.py

```diff
@@ -320,7 +320,52 @@ def aggregate(self, arg, *args, **kwargs):
     agg = aggregate
 
     _shared_docs['sum'] = dedent("""
-    %(name)s sum""")
+    Calculate %(name)s sum of given DataFrame or Series.
+
+    Parameters
+    ----------
+    *args
+        Under Review.
+    **kwargs
+        Under Review.
+
+    Returns
+    -------
+    Series or DataFrame
+        Like-indexed object containing the result of function application
+
+    See Also
+    --------
+    Series.%(name)s : Calling object with Series data
+    DataFrame.%(name)s  : Calling object with DataFrame data
+    Series.sum  : Equivalent method for Series
+    DataFrame.sum   : Equivalent method for DataFrame
+
+    Examples
+    --------
+    >>> s = pd.Series([1, 2, 3, 4, 5])
+    >>> s.rolling(3).sum()
+    0     NaN
+    1     NaN
+    2     6.0
+    3     9.0
+    4    12.0
+    dtype: float64
+    >>> s.expanding(3).sum()
+    0     NaN
+    1     NaN
+    2     6.0
+    3    10.0
+    4    15.0
+    dtype: float64
+    >>> s.rolling(3, center=True).sum()
```

Comment:
I think this is a great example, but maybe add one line or comment for what to expect with `center=True`

Reply:
Thank you. I have added one line of explanation stating that how this argument helps.